webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  mode: 'development',
+
+  entry: {
+    components: './presets/components.ts',
+  },
+
+  devtool: 'source-map',
+
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist')
+  },
+
+  plugins: [new webpack.ProgressPlugin()],
+
+  module: {
+    rules: [{
+      test: /.(ts|tsx)?$/,
+      loader: 'ts-loader',
```

Comment:
Also same feedback from logic, do we want to look into `awesome-typescript-loader`?

Reply:
Updated.